lib/rules/role-img-alt.json

```diff
@@ -1,6 +1,7 @@
 {
 	"id": "role-img-alt",
-	"selector": "[role='img']:not(svg):not(img):not(area):not(input):not(object)",
+	"selector": "[role='img']:not(img):not(area):not(input):not(object)",
```

Comment:
Should add a test to this rules integration test to ensure svg elements aren't captured by this rule now.

Reply:
gd catch.